docs/rackhd/api-usage-examples.rst

```diff
@@ -0,0 +1,102 @@
+Rest-2.0 API examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Firmware update Example using SKU Pack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This example provides instructions on how to flash a BMC image on a Quanta (node) using SKU Pack.
+
+1. Wait for discovery to complete and Get nodes to check if node has been discovered successfully
+
+     **Get Nodes**
+     ::
+     
+         GET /api/2.0/nodes
+     ::
+     
+         curl <server>/api/2.0/nodes
+
+
+2. Post the obm settings if they don't already exist for the node. An example on how to do this is shown in Section 7.1.8.1 here http://rackhd.readthedocs.io/en/latest/tutorials/vagrant.html#adding-a-sku-definition Section 7.1.8.1
+
+
+3. The firmware files need to be built into a SKU package
+  
+     **Build SKU Package**
+     ::
+     
+         $ ./build-package.bash <sku_pack_directory> <subname>
+
+   The new tarball is built in a folder called tarballs within the same directory 
+    
+     ::
+        
+        $ /tarballs 
+          sku_pack_directory_subname.tar.gz
+          
+4. The SKU package that was built needs to be registered            
+
+     **POST the tarball**
+     ::
+     
+        curl -X POST --data-binary @tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack
```

Comment:
@uppalk1 `/api/common` has been deprecated.  We should use `/api/current` instread